README.md

```diff
@@ -1,2 +1,155 @@
 # Set_up_a_-Hello_Microverse-_project
-To practice the skills that I just learned. 
+To practice the skills that I just learned from Microverse.
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <h3><b>Set up a "Hello Microverse" project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [HTML&CSS](#html-tech)
+    - [Linters & .gitignore](#lin-git)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
```

Comment:
Well done on using the README template 💯 but make sure that no sections are removed from the `Table of Contents` and that all the sections are referenced in the README body 📖 .  Remember, only the `FAQ` is optional 🤷🏽 , you can also feel free to remove the `Live version` as it does not apply to this project. 😉 Take a look at the [readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) 🔗

Reply:
Hi @didierganthier 
Thank for the review
I have a doubt about creating a new branch, I will be create a new branch and merge this on that new branch and after that make a new pull request, is it correct?

Reply:
Also about the commit messages is an aclaration for my next projects right? 
And about the README file, it doesn't need changes, does it? or have I to change it? 
I make this questions because is my first project and I want to undertand this well.
I appreciate your help and comments, thank you.

Reply:
Hello @AndreaM2429 you just create a new branch from this one and you open a PR from that new branch to the `main` branch, make sure to add the link of that PR in the new PR description. 

For the commit messages, this is related to the next commits for this project and this should be taken into account for the next projects as well.

For the README, your README is lacking several required sections, all the sections in the template are required except for the `FAQ`, the `Live Demo` can also be removed as it's not relevant for this project.